### DISCUSSION TITLE
Phase 2 (Post MVP) Brainstorming

### DISCUSSION THREAD

**Discussion by intelagense** [Together-100Devs/Together#94] [General]
I am placing some ideas here for the next phase and beyond. 
Feel free to add your own or discuss.

- The client caches data from MongoDB +3 to +6 months in either direction from the current calendar view to reduce rendering time when cycling back and forth through the months. This will avoid waiting on rapid network requests or downloading the entire collection to render the events.

- Cancel button on event details (can be open to the event creator)

- Report/flag button on events (for events that were entered incorrectly or no longer maintained. This can be sent to an admin or maintainer in a report)

**Comment by Caleb-Cohen** [comment]
Some sort of notification of events about to start would be cool. I would not be surprised if there are discord bots that could handle that.
